resident/resident-service/src/main/java/io/mosip/resident/constant/ServiceType.java

```diff
@@ -34,4 +34,14 @@ public static Optional<ServiceType> getServiceType(String serviceTypeString) {
         }
         return Optional.empty();
     }
+    
+    public static String getServiceTypeFromRequestType(RequestType requestType) {
```

Comment:
Return Optional<String> and use Optional.empty() where null value is returned.

Reply:
changes done